flow/DraftJSFlowStub.js.flow

```diff
@@ -4,4 +4,5 @@ declare export var ContentState: any
 declare export var Editor: any
 declare export var CompositeDecorator: any
 declare export var Modifier: any
+declare export var SelectionState: any
```

Comment:
One thing to note is that this stub is extremely limited to the Flow types that ship with Draft by default. We should probably switch to the real types and fix the corresponding errors.